tools/testbench/topology_ipc4.c

```diff
@@ -330,10 +331,84 @@ int tb_match_audio_format(struct testbench_prm *tp, struct tplg_comp_info *comp_
 	return 0;
 }
 
+static int tb_set_pin_formats(struct tplg_comp_info *comp_info,
+			      struct ipc4_base_module_cfg *base_cfg,
+			      struct ipc4_base_module_cfg_ext *base_cfg_ext, int pin_type)
+{
+	struct sof_ipc4_pin_format *formats_array;
+	struct sof_ipc4_pin_format *pin_format_item;
+	struct sof_ipc4_pin_format *pin_format;
+	struct sof_ipc4_pin_format *base_cfg_ext_pin_formats =
+		(struct sof_ipc4_pin_format *)base_cfg_ext->pin_formats;
+	int format_list_count;
+	int num_pins;
+	int i, j;
+	int pin_format_offset = 0;
+
+	if (pin_type == 0) {
```

Comment:
I find these make it harder to read, can you add a enum for 0/1?

Reply:
Yep, done!